src/deadline/job_attachments/upload.py

```diff
@@ -163,6 +165,8 @@ def upload_assets(
             job_attachment_settings: The settings for the job attachment configured in Queue.
             progress_tracker: Optional progress tracker to track progress.
             manifest_name_suffix: Suffix for given manifest naming.
+            manifest_metadata: File metadata for given manifest to be uploaded.
+            manifest_file_name: Optional file name for given manifest to be uploaded, otherwise use default name.
```

Comment:
nit: can we add details on what the `default` name is and where it comes from? This is the one derived from the hash of the asset root?

Reply:
Do you think this is blocking? I'd like to take it in a follow-up if possible.